ecl/resource_ecl_provider_connectivity_tenant_connection_v2.go

```diff
@@ -0,0 +1,461 @@
+				Optional:      true,
+				MaxItems:      1,
+				ConflictsWith: []string{"attachment_opts_vna"},
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"segmentation_type": &schema.Schema{
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"segmentation_id": &schema.Schema{
+							Type:     schema.TypeInt,
+							Optional: true,
+						},
+						"fixed_ips": &schema.Schema{
+							Type:     schema.TypeList,
+							Optional: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"ip_address": &schema.Schema{
+										Type:         schema.TypeString,
+										Optional:     true,
+										ValidateFunc: validation.SingleIP(),
+									},
+									"subnet_id": &schema.Schema{
+										Type:     schema.TypeString,
+										Optional: true,
+									},
+								},
+							},
+						},
+						"allowed_address_pairs": &schema.Schema{
+							Type:     schema.TypeList,
+							Optional: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"ip_address": &schema.Schema{
+										Type:         schema.TypeString,
+										Optional:     true,
+										ValidateFunc: validation.SingleIP(),
+									},
+									"mac_address": &schema.Schema{
+										Type:     schema.TypeString,
+										Optional: true,
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			"attachment_opts_vna": &schema.Schema{
+				Type:          schema.TypeList,
+				Optional:      true,
+				MaxItems:      1,
+				ConflictsWith: []string{"attachment_opts_server"},
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"fixed_ips": &schema.Schema{
+							Type:     schema.TypeList,
+							Optional: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"ip_address": &schema.Schema{
+										Type:         schema.TypeString,
+										Optional:     true,
+										ValidateFunc: validation.SingleIP(),
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			"name_other": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"description_other": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"tags_other": &schema.Schema{
+				Type:     schema.TypeMap,
+				Optional: true,
+			},
+			"tenant_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"tenant_id_other": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"network_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"port_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"status": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceProviderConnectivityTenantConnectionV2Create(d *schema.ResourceData, meta interface{}) error {
```

Comment:
The same of `resourceProviderConnectivityTenantConnectionRequestV2Create` applies to this function. You need to call update request as well.

Reply:
Yes, confirm.

Reply:
Same as this answer.
https://github.com/nttcom/terraform-provider-ecl/pull/46#discussion_r379251432